stdlib/string.gr

```diff
@@ -1399,23 +1514,28 @@ export let forEachCodePoint = (fn: (Number) -> Void, str: String) => {
     // avoid heap allocations. `getCodePoint` will throw
     // MalformedUnicode exception for values exceeding this limit.
     let codePoint = getCodePoint(ptr)
+    Memory.incRef(WasmI32.fromGrain(fn))
```

Comment:
incref'd in a loop, but only decref'd once outside of the loop?

Reply:
Same as above; it will `decRef` itself per the calling convention